lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -336,6 +336,7 @@ dictionary Config {
     string? cache_dir;
     u64? zero_conf_max_amount_sat;
     sequence<ExternalInputParser>? external_input_parsers = null;
+    u16? updated_fee_auto_acceptance_threshold_percent;
```

Comment:
Is the idea for it to be a static threshold or one relative to the original estimate? Example: accept any rate below `threshold_sat_per_vbyte` **vs** accept  when fee rate is below original `original_fee_rate_estimate`+ `threshold_sat_per_vbyte` 

@kingonly Could you elaborate on why either of the former would be preferable? IMO, it would be easier for a user to understand and choose a maximum relative increase of an initially estimate fee. Example: accept if the quoted fee is below `1.2 * original_fee_estimate`

Reply:
Because I don't think percentage work in the context of on-chain fees. Users understand sats/vbyte. For example if you tell them that they can broadcast up to +/-5 sats/vbyte, they would understand it. Percentages make less sense because users would pat 200, 400, 600, 1000 sats to broadcast 100k transaction. The alternative would an absolute max fee budget, but again,I think the UX is not as good.
 
Maybe look into the sweep implementation in Lightning to get some inspiration for automatic broadcasting.

Reply:
> 1.2 * original_fee_estimate

That's actually what I meant by percentage which is 20% tolerance in this case.

Reply:
I changed the new config field to a sat/vbyte leeway in [97d6ca6](https://github.com/breez/breez-sdk-liquid/pull/615/commits/97d6ca687874c66d58b714960d365c33ed8d694f).